### ISSUE TITLE
Requests to paginated views of the homepage should redirect to the homepage

### ISSUE THREAD

**Issue by jonoalderson** [WordPress/Learn#2480]
URLs that request paginated views of the homepage, like https://learn.wordpress.org/page/4/ are publicly available but don't return a paginated state. This represents an SEO and analytics headache, which we should clean up.

All such requests should redirect to the homepage with a `301` HTTP status

**Comment by kaitohm** [comment]
//dev

**Comment by jonathanbossenger** [comment]
@adamwoodnz is this something we can fix now, or should we think about it for the redesign?

**Comment by adamwoodnz** [comment]
I think it can be fixed anytime, asap

**Comment by Mayur3795max** [comment]
You're right — paginated URLs like https://learn.wordpress.org/page/4/ currently return a 200 status without meaningful content, which isn't ideal for SEO or analytics.

We should implement a 301 redirect for these URLs to point back to the homepage (https://learn.wordpress.org/). This will help:

* Prevent indexing of empty or duplicate pages

* Consolidate link equity

* Improve crawl efficiency

I'll look into adding the necessary redirect logic to handle this gracefully.
